userguide/drawing_classifier/data-preparation.md

```diff
@@ -15,11 +15,11 @@ First, we will download all the data points in the "Quick, Draw!" dataset for
 the "square" and "triangle" classes -- around 120,000 examples each, 
 both as bitmaps and as stroke-based drawings.
 
-Open a new terminal window and run the following commands:
+Open a new terminal window, go to your working directory and run the following commands:
 
 ```
-$ mkdir -p ~/Downloads/quickdraw
-$ cd ~/Downloads/quickdraw
+$ mkdir -p quickdraw
```

Comment:
There's no need for the `-p` flag anymore.

Reply:
Fixed.